sonar-plugin-api/src/main/java/org/sonar/api/utils/DateUtils.java

```diff
@@ -54,6 +55,10 @@ public static String formatDate(Date d) {
     return d.toInstant().atZone(ZoneId.systemDefault()).toLocalDate().toString();
   }
 
+  public static String formatDate(Date d, TimeZone timeZone) {
```

Comment:
I'm not sure we should add more stuff in the plugin API. It's always a mess to maintain. What about keeping it private?

Reply:
👍